packaging/greenboot/microshift-pre-rollback.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e -o pipefail
+
+SCRIPT_NAME=$(basename $0)
+
+# Set the exit handler to log the exit status
+trap 'script_exit' EXIT
+
+# The script exit handler logging the FAILURE or FINISHED message depending
+# on the exit status of the last command
+#
+# args: None
+# return: None
+function script_exit() {
+    [ "$?" -ne 0 ] && status=FAILURE || status=FINISHED
+    echo $status
+}
+
+#
+# Main
+#
+
+# Exit if the current user is not 'root'
+if [ $(id -u) -ne 0 ] ; then
+    echo "The '${SCRIPT_NAME}' script must be run with the 'root' user privileges"
+    exit 0
+fi
+
+echo "STARTED"
+
+# Dump GRUB boot variables and ostree status
```

Comment:
The grub settings affect the execution of the script. It only works when "boot_counter=0", so I thought it would be good to have these settings in the logs. 
In general, both greenboot-related scripts dump these settings because this info is critical when troubleshooting

Reply:
OK, a more detailed comment about that would be good.